### PR TITLE
3.0.0/upgrade rn version to 74

### PR DESCRIPTION
**Pull request by MusiikoAS**

### CODE REVIEW

File: src/Options.tsx [hunk 1]
```diff
@@ -12,65 +11,10 @@ export type NavigationBarConfig = {
 
 export type StatusBarConfig = {
     lightStatusBar?: boolean,
-    /**
```
Comment (susslik) [inline src/Options.tsx:15]: That was done in order to show the information to the user, pls check the screenshot bellow:
Screenshot 2024-05-10 at 14 23 30

But with the updated implementation - it's only our internal commnent:
Screenshot 2024-05-10 at 14 23 41

File: src/Options.tsx [hunk 2]
```diff
@@ -12,65 +11,10 @@ export type NavigationBarConfig = {
...
-
-export type ConfigOptionName = "isS2S" | "userId" | "sessionUid" | "dataPoints";
-export type StylingOptionName = "title" | "language" | "navigationBar" | "statusBar";
-export type NavigationBarOptionName = "backgroundColor" | "buttonsColor" | "titleColor" | "hasShadow";
-export type StatusBarOptionName = "lightStatusBar";
-
-/**
- * @deprecated Please, use `setSessionID` and `setUserID` functions instead
- */
-export type ConfigOptionTypes = {
```
Comment (susslik) [inline src/Options.tsx:36]: Let's make similar to the methods - remove all from the inside but leave the type declaration with @unsupported annotation and the description of what to use instead

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
+##[3.0.0] 2024-05-02
+
+### Changed
+- React Native version updated to 0.74.0
+- `init` deprecated method - only method declaration lefted;
```
Comment (susslik) [inline CHANGELOG.md:5]: Let's update the description a bit (for all the things bellow):
`init` - no longer supported;

**Review by susslik — CHANGES_REQUESTED** [review]
Hey @MusiikoAS  - not bad, however I have found a few possible improvements, could you take a look pls?

File: android/build.gradle [hunk 4]
```diff
@@ -9,10 +9,10 @@
 apply plugin: 'com.android.library'
 apply plugin: 'maven-publish'
 
-def DEFAULT_COMPILE_SDK_VERSION = 33
-def DEFAULT_BUILD_TOOLS_VERSION = '33.0.0'
-def DEFAULT_MIN_SDK_VERSION = 21
-def DEFAULT_TARGET_SDK_VERSION = 33
+def DEFAULT_COMPILE_SDK_VERSION = 34
```
[the remark this answers is not in the document]
Comment (MusiikoAS) [reply android/build.gradle]: done.

File: src/Options.tsx (hunk 1 above)
Comment (MusiikoAS) [reply src/Options.tsx:15]: yes, you are right - fixed now.

File: src/Options.tsx (hunk 2 above)
Comment (MusiikoAS) [reply src/Options.tsx:36]: done

File: CHANGELOG.md (hunk 3 above)
Comment (MusiikoAS) [reply CHANGELOG.md]: done

**Review by susslik — APPROVED** [review]
Great job @MusiikoAS

**Pull request merged**